orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/manifests/BakeManifestContext.java

```diff
@@ -36,19 +38,23 @@
   // Lombok (1.16.20) that
   // Jackson can use to deserialize.
   public BakeManifestContext(
-      @JsonProperty("inputArtifacts") List<CreateBakeManifestTask.InputArtifactPair> inputArtifacts,
+      @Nullable @JsonProperty("inputArtifacts")
```

Comment:
If the constructor can accept null for these two fields, it would make sense to add `@Nullable` to the fields themselves (as they'll end up null in some cases).